internal/service/dns_config/model_config_forward_zone_config.go

```diff
@@ -0,0 +1,103 @@
+package dns_config
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+
+	"github.com/infobloxopen/bloxone-go-client/dns_config"
+
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/flex"
+)
+
+type ConfigForwardZoneConfigModel struct {
```

Comment:
i don't think this model is used. you can remove it.

Reply:
Resolved